backend/src/main/java/hu/flowacademy/munkalap/service/UserService.java

```diff
@@ -0,0 +1,60 @@
+package hu.flowacademy.munkalap.service;
+
+import hu.flowacademy.munkalap.entity.User;
+import hu.flowacademy.munkalap.enumCustom.Kind;
+import hu.flowacademy.munkalap.exception.WorksheetUserException;
+import hu.flowacademy.munkalap.repository.UserRepository;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+
+    @Autowired
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public User saveUser(User user) throws WorksheetUserException {
+        Objects.requireNonNull(user);
```

Comment:
Ez is mehet akár a `validateUser`be, mivel ez is egyfajta validáció

Reply:
Vagy miért nem használod inkább a lombok `@NonNull` annotációt(https://projectlombok.org/features/NonNull)